challenges/objects-arrays.js

```diff
@@ -7,28 +7,50 @@
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
-
+const tyrannosaurus = {
+  name :"Tyrannosaurus",
+  diet : "carnivorous",
+  weight : 7000,
+  length : 12,
+  period : "Late Cretaceious",
+  roar : function(){
+    return "RAWERSRARARWERSARARARRRR!";
+  }
+}
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
-
+const stegosaurus = {
+  name : "Stegosaurus",
+  diet : "herbivorous",
+  weight : 2000,
+  length : 9,
+  period : "Late Jurassic",
+}
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
-
+const velociraptor = {
+  name : "Velociraptor",
+  diet : "carnivorous",
+  weight : 15,
+  length : 1.8,
+  period : "Late Cretaceious",
+}
 // Using your dinosaur objects, log answers to these questions:
 
 // How much did tyrannosaurus weigh?
-console.log();
+console.log(`${tyrannosaurus.name} weighed ${tyrannosaurus.weight}kg.`);
 
 // What was the diet of a velociraptor?
-console.log();
+console.log(`${velociraptor.name} are ${velociraptor.diet}.`);
 
 // How long was a stegosaurus?
-console.log();
+console.log(`${stegosaurus.name} are ${stegosaurus.length}m long.`);
 
 // What time period did tyrannosaurus live in?
-console.log();
+console.log(`${tyrannosaurus.name} came from the ${tyrannosaurus.period} period.`);
 
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
-console.log();
+
+console.log(tyrannosaurus.roar());
 
 
```

Comment:
Good understanding of objects. Although unnecessary, it's really great that you call out the name of each dinosaur.